src/generic/clipboard/paste-component/index.tsx

```diff
@@ -28,6 +28,7 @@ const PasteComponent = ({
         onMouseLeave={() => handlePopoverToggle(false)}
         onFocus={() => handlePopoverToggle(true)}
         onBlur={() => handlePopoverToggle(false)}
+        {...props}
```

Comment:
What is this change doing?

Reply:
The paragon `OverlayTrigger` passes some props to this component necessary for its positioning.
Without it, the popover appears on the top of the screen.

Reply:
Ah, thanks for figuring that out!